src/client/components/UserEditor.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import { User } from '../../shared/User';
+import { remult } from '../utils/api-facade';
+import { Box } from '@mui/material';
+import { ErrorInfo } from 'remult';
+
+const userRepo = remult.repo(User);
+
+interface IProps {
+    user: User;
+    onClose: () => void;
+    onSaved: (user: User) => void;
+    create?: boolean;
+}
+export const UserEditor: React.FC<IProps> = ({ user, onClose, onSaved, create }) => {
+    const [data, setData] = React.useState<Partial<User>>(user);
+    const [errors, setErrors] = React.useState<ErrorInfo<User>>(null);
+    const handleClose = () => {
+        onClose();
+    };
```

Comment:
Unless you specifically wanna ignore the arguments which the event handler is called with, this wrapper function is useless.

Reply:
ok